apps/lend/src/components/AlertLoanSummary/components/SummarySelfLiquidate.tsx

```diff
@@ -18,10 +19,34 @@ const SummarySelfLiquidate = ({
   const { debt: stateDebt = '0', collateral: stateCollateral = '0', borrowed: stateBorrowed = '0' } = userState ?? {}
   const { borrowed: walletBorrowed = '0' } = userWallet ?? {}
 
-  const amountNeededFromWallet = +stateDebt - +stateBorrowed
-  const walletAmount = amountNeededFromWallet - +walletBorrowed <= 0 ? amountNeededFromWallet : +walletBorrowed
-  const remainingBalance = +stateDebt - +stateBorrowed - +walletAmount
-  const minWidth = '180px'
+  const walletAmount = useMemo(() => {
+    const amountNeeded = +stateDebt - +stateBorrowed
+    const amountNeededFromWallet = amountNeeded > 0 ? amountNeeded : 0
+
+    return amountNeededFromWallet > 0
+      ? BN(amountNeededFromWallet).isGreaterThan(walletBorrowed)
```

Comment:
Can we gradually move away from using the BigNumbers library as deprecated? Since the BigInt type is supported by native JS and the BN library should be eliminated in the future.